runtimes/lemonade/start.sh

```diff
@@ -0,0 +1,197 @@
+#!/bin/bash
+set -e
+
+# Lemonade Server Startup Script
+# This script starts the Lemonade inference server with OpenAI-compatible API
+# Designed to be extensible for future runtime additions (vLLM, etc.)
+
+# Configuration with sensible defaults
+# These can be overridden by environment variables or project config
+LEMONADE_PORT="${LEMONADE_PORT:-11534}"
+LEMONADE_HOST="${LEMONADE_HOST:-127.0.0.1}"
+LEMONADE_BACKEND="${LEMONADE_BACKEND:-onnx}"  # Default to ONNX (works on all systems)
+LEMONADE_CONTEXT_SIZE="${LEMONADE_CONTEXT_SIZE:-32768}"  # Default context size
+LEMONADE_MODEL="${LEMONADE_MODEL:-}"  # Optional: pre-load a specific model
+
+# Try to read config from project llamafarm.yaml if available
+# This allows the startup script to respect project configuration
+# Try multiple possible locations for llamafarm.yaml
+CONFIG_FILE=""
+if [ -f "../../llamafarm.yaml" ]; then
+    CONFIG_FILE="../../llamafarm.yaml"
+elif [ -f "../llamafarm.yaml" ]; then
+    CONFIG_FILE="../llamafarm.yaml"
+elif [ -f "llamafarm.yaml" ]; then
+    CONFIG_FILE="llamafarm.yaml"
+fi
+
+if [ -n "$CONFIG_FILE" ]; then
```

Comment:
This approach does not handle a multi-model approach correctly. it will always return the following config values for the last model in the runtime.models list